PyFlyt/models/vehicles/fixedwing/cf2.dae

```diff
@@ -0,0 +1,1867 @@
+<?xml version="1.0" encoding="utf-8"?>
+<COLLADA xmlns="http://www.collada.org/2005/11/COLLADASchema" version="1.4.1">
+  <asset>
+    <contributor>
+      <author>Lakshman Kumar</author>
```

Comment:
Rename this file to something more descriptive. cf2 stands for crazyflie2 which refers to the quadx model.